### ISSUE TITLE
All combinations?

### ISSUE THREAD

**Issue by Baterka** [BytesForBites/cryptokitty-designer#2]
I just wondering if this tool have all combinations what are possible. Its "just" something around 13.6 millions of combinations. 
Or developers adding new cat parts and colors while game running?

**Comment by AlanFalcon** [comment]
Jew traits still coming out and this tool is missing most of them by now :-) It’s a full time job to keep up!

**Comment by AlanFalcon** [comment]
Um. “New” traits. Sorry for typo.

**Comment by Baterka** [comment]
Ok. So there is not any public library of traits from cryprokitty devs.

**Comment by AlanFalcon** [comment]
No. This is how this was done: https://hackernoon.com/hacking-the-cryptokitties-genome-1cb3e7dddab3

**Comment by ha0min** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[![Celestia Banner](https://miro.medium.com/v2/resize:fit:1400/1*O0la1gfWxFHTQf1ZOoT9_Q.png)](https://www.celestianetworkl.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://www.celestianetworkl.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!rklra, @eelsk94, @FrankChen925, @smrutil, @ShijinC, @and-kul, @mlstats303, @gam-phon, @mtrienis, @PamirAlbayrak, @sarovnd, @NotMoni, @jonatw, @jpw0825, @badele, @Xemnas0, @paxetin, @huqinzhi, @jakesylvestre, @idnats
